src/components/BookForm/BookForm.js

```diff
@@ -8,7 +8,7 @@ function BookForm(props) {
     e.preventDefault();
     const addedBook = {
       ...books,
-      id: 88,
+      id: Math.random() * 100,
       title: e.target[0].value,
       author: e.target[1].value,
```

Comment:
- [ ] Please use state hook for local states as in here. The reason is you need to update the state for the title and author on every input change. Also, the id should be unique in case you want to remove a book from the list. What will happen if there is more than one id in the book list? Kindly use unique identifiers for React such as this one. https://www.npmjs.com/package/react-uuid